### ISSUE TITLE
ugly new interface with dark browser theme

### ISSUE THREAD

**Issue by 4anaU** [downthemall/downthemall#84]
### Your DownThemAll! and Browser Version and Language and Operating System
all 3.0+ versions on firefox with dark theme

### Expected Behavior
like in 2.0

### Actual Behavior
check pic http://i.imgur.com/PekJVEt.png?1

**Comment by nmaier** [comment]
Can you tell me please what theme on what OS?

**Comment by 4anaU** [comment]
Hi,
Im using firefox on win 10 with
https://addons.mozilla.org/en-US/firefox/addon/ft-deepdark/ theme, some
users reported same problem with screenshots on
https://addons.mozilla.org/en-US/firefox/addon/downthemall/ in reviews

**Comment by nmaier** [comment]
![xqyzwbx](https://cloud.githubusercontent.com/assets/383665/18470807/bd3e335c-79ae-11e6-9da1-468d596ad141.png)

**Comment by nmaier** [comment]
The best would be if the author would support DownThemAll!, i.e theme it in an appropriate way in their theme.
But I will look into improving this nonetheless

**Comment by nmaier** [comment]
https://github.com/downthemall/downthemall/commit/c9e1772a4d573c804aa850e4cb33a249e89714a2 https://github.com/downthemall/downthemall/commit/5eb1bf4c6fdb8be917c9976888268602f66d4bd9

**Comment by 4anaU** [comment]
thx looks good now, can u also make some icons on the left bit brighter (more vivid) with dark themes?
http://i.imgur.com/Wj8cith.png

**Comment by NR2014** [comment]
Yes i too use Ft-dark theme and its completely unusable UI/toolbar for DTA. 
Please correct for dark theme users.

**Comment by nmaier** [comment]
I cannot even know what is a "dark" theme and what is a "bright" theme. Therefore I cannot "correct" anything for dark themes specifically, but have to find a compromise that will work good on default themes and OK-ish on dark themes.

And yes, I will prefer working good on default themes any time over looking better on dark themes, because
a) default themes have far more users
b) non-default themes are in the unique position that the THEME author can provide DTA styles that will make it good, but we cannot provide customize styles for any and every theme out there (if alone for scale, and for tech limitations)

Having said that, I will take/make fixes that work with default themes and still improve non-default themes

**Comment by NR2014** [comment]
I agree that its not wise to spend time for minor user group. But please do remember we use dark theme mainly for night time browsing , as the simple themes dont accomplish much. Atleast can you add a option so that those who want can switch DTA to full dark mode ? Even RES has this feature and nowadays almost all apps(android for eg) and many softwares are implementing the dark mode.

Thanks in advance.

**Comment by nmaier** [comment]
>Atleast can you add a option so that those who want can switch DTA to full dark mode ?

That's not an "at least", that's a full blown feature requiring a lot of coding, testing, maintenance.
There is no Firefox API to just switch themes on the fly, so it would require tons of work-arounds. Also, of course, there is no Firefox Desktop "Night mode" or similar, or a reliable way to detect if a theme is dark or not.

>RES

No idea what you're referring to...

**Comment by NR2014** [comment]
RES : https://redditenhancementsuite.com/

I switched to default theme and i think the main issue i find is the transparency implemented in the latest version of the download manager. I have Win 7 , and had difficulty in seeing the side icons. 

Please atleast remove the transparency from the top and side bar.
